### DISCUSSION TITLE
hugo v0.131 WARN deprecated: .Site.Author

### DISCUSSION THREAD

**Discussion by wildserver** [razonyang/hugo-theme-bootstrap#1041] [Q&A] [answered]
Here is the translated message:

---

Hi,

I updated Hugo to version v0.131, and during site generation, a warning appears:

```
WARN  deprecated: .Site.Author was deprecated in Hugo v0.124.0 and will be removed in a future release. Use taxonomies instead.
```

I haven't noticed any issues, but could you find some time to address this warning?

Thank you for your work. 😊

Best regards.

**Comment by razonyang** [comment]
Will take closer a look into it.

**Comment by razonyang** [comment, marked as answer]
Done, see the [v1.5.8](https://github.com/razonyang/hugo-theme-bootstrap/releases/tag/v1.5.8) release notes to migrate.

**Reply by wildserver** [reply]
Excellent work. Thank you.
As always, everything is fast and high quality.
Respect.

**Reply by razonyang** [reply]
You're welcome.
